### DISCUSSION TITLE
Self hosting

### DISCUSSION THREAD

**Discussion by han9over** [mhdzumair/MediaFusion#152] [Q&A]
Looking to host this locally inside network on a raspberry pi. do we need ssl for internal hosting? how different are the instructions to self host on a device like pi?

**Comment by mhdzumair** [comment]
its same as other device. but the dns you may need to setup to access outside of pi. there are some users already using it. you can join the tg to ask help from them

**Reply by han9over** [reply]
I'm sorry, what is TG? is it telegram? If so, i don't have one. do you know if we can install it without HTTPS?

**Reply by mhdzumair** [reply]
Without https is not possible. The posters wont work without https.

**Reply by han9over** [reply]
thanks. will try to see I can figure that out.

**Reply by jas0npc** [reply]
did this get fiqured out?
